src/config/constants/pools.ts

```diff
@@ -1,8 +1,22 @@
+import { VaultKey } from 'state/types'
 import { serializeTokens } from './tokens'
 import { SerializedPoolConfig, PoolCategory } from './types'
 
 const serializedTokens = serializeTokens()
 
+export const vaultPoolConfig = {
+  [VaultKey.CakeVault]: {
+    name: 'Auto CAKE',
+    description: 'Automatic restaking',
+    autoCompoundFrequency: 5000,
+  },
+  [VaultKey.IfoPool]: {
+    name: 'IFO CAKE',
+    description: 'Stake CAKE to participate in IFO',
+    autoCompoundFrequency: 5000, // TODO: update IFO compound frequency
```

Comment:
@Chef-Cheems What's the `autoCompoundFrequency` for this pool?

Reply:
Lets say 2000.
It is not exact, just approximate. Depends how many times users stake or harvest.
To be honest the difference in shown APY would be tiny anyway between 2000 and 5000. The law of diminishing returns is kinda tough on the compounding, after certain compounding cycles it barely makes a difference.